core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -361,6 +361,7 @@ abstract class RDD[T: ClassTag](
       readCachedBlock = false
       computeOrReadCheckpoint(partition, context)
     }) match {
+      // Block hit.
```

Comment:
not related to the change?

Reply:
True, I can back it out if needed, but I was reading this code path and thought this was a useful comment to add while I was in here.